src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs

```diff
@@ -224,6 +225,44 @@ static int GetNdkApiLevel(string androidNdkPath, string androidApiLevel, Android
 
 			var nativeLibs = new List<string> ();
 
+			CancellationTokenSource cts = null;
```

Comment:
Could you please place this into a separate method? Help keep method size "reasonable"?